pkg/rest/request/request.go

```diff
@@ -46,6 +55,8 @@ func (c Client) Send(ctx context.Context, route, method string, body, responsePt
 		bytes.NewBuffer(buf),
 	)
 
+	logger.Info("Sending request to:" + c.routeToURL(route))
```

Comment:
Remove this if it was for debugging purposes

Reply:
I'll leave it as debug